apis/fluentbit/v1alpha2/plugins/common_types.go

```diff
@@ -0,0 +1,19 @@
+package plugins
+
+import "github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins/params"
+
+// +kubebuilder:object:generate:=true
+
+type CommonParams struct {
+
+	// Alias for the plugin
+	Alias string `json:"alias,omitempty"`
+}
+
+func (c *CommonParams) ParseParams(sl SecretLoader) (*params.KVs, error) {
```

Comment:
The `sl SecretLoader` is never used here, we can pass `kvs` as param here instead of `sl` and we'll needn't put NewKVs in ParseParams. 

Also I think `ParseParams` can be changed to `AddCommonParams`